source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md

```diff
@@ -164,7 +166,7 @@ You can clean up the index.html file when you are done by running: `sudo rm inde
 
 Make sure the README.md is in pwd. Then enter:
 
-```
+```Spark-Shell
```

Comment:
Afaik, this is Scala code, which is entered into Spark Shell. We can use 'Scala' for proper syntax highlight, but I wanted to sow it is done inside Spark Shell, not shell or IDE.